src/test/ui/parser/issue-62973.stderr

```diff
@@ -13,9 +13,11 @@ error: expected one of `,` or `}`, found `{`
   --> $DIR/issue-62973.rs:6:25
    |
 LL | fn p() { match s { v, E { [) {) }
-   |                -        ^ expected one of `,` or `}` here
-   |                |
-   |                while parsing this struct
+   |        -       -       -^ expected one of `,` or `}` here
+   |        |       |       |
+   |        |       |       help: `}` may belong here
+   |        |       while parsing this struct
```

Comment:
Pre-existing... but ouch; it thinks it is parsing a struct. :(

The new help note here is leading astray as well.

Reply:
Yeah, this is not good. I didn't dig deeper when I fixed this, as it came straight out of a fuzzer, but I am intrigued at how we end up parsing a struct here :-/

Reply:
Ah; given that it is fuzzer output I would say that it is OK.